presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkQueryExecutionFactory.java

```diff
@@ -698,7 +785,27 @@ else if (executionException instanceof PrestoSparkExecutionException) {
                     result.add(unmodifiableList(columns));
                 }
             }
-            return result.build();
+            List<List<Object>> results = result.build();
+
+            // Based on com.facebook.presto.server.protocol.Query#getNextResult
+            OptionalLong updateCount = OptionalLong.empty();
```

Comment:
This is updateRowCount.

Reply:
Yeah I am wondering whether this class should still be called `PrestoSparkQueryInfo` -- it's based on `QueryInfo` at beginning but now we have remodel it to more like `QueryResults`.. 😄 

Not QueryResults in Presto classic implements two interface: `QueryData` and `QueryStatusInfo`. Does it resembles `QueryStatusInfo` more now?

Reply:
Yeah. Let me rename it to `PrestoSparkQueryStatusInfo`. I would love to keep the `updateCount` as is to keep it binary compatible with `QueryResults`, so we can reuse the tools that process it.